src/Uno.UI/UI/Xaml/Shapes/ArbitraryShapeBase.iOSmacOS.cs

```diff
@@ -300,10 +300,26 @@ protected override Size MeasureOverride(Size size)
 			var path = GetPath();
 			if (path == null)
 			{
-				return default(Size);
+				return default;
 			}
 			var bounds = path.PathBoundingBox;
 
+			if (bounds.IsEmpty)
+			{
+				return default;
+			}
+
+			// On iOS 11, the origin (X, Y) of bounds could be infinite, leading to strange results.
+			if (nfloat.IsInfinity(bounds.X))
+			{
+				bounds.X = 1;
```

Comment:
Not sure it matters but 1 seems odd no? Isn't it better to set 0 ?

Reply:
Same comment

Reply:
Zero would work the same way here. Anyway when the crash is happening the control is not visible.

I'll switch it to zero.